### PR TITLE
Pump commands, integration

### PR DESCRIPTION
**Pull request by pkubanek**
- PumpWidget
- Power button,..
- ColoredButton class
- TopicStatusLabel
- Correct units

### CODE REVIEW

File: python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+class FieldButton(ColoredButton):
+    """Display single field in topic.
+
+    Parameters
+    ----------
+    field : `str`
+    inactive : `(str, QColor)`
+    active : `(str, QColor)`
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py:35]: Description?

File: python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        self.fields = fields
+        self.window = None
+
+        self.clicked.connect(self._clicked)
+        signal.connect(self.data)
+
+    def _clicked(self):
+        if self.window is None:
+            self.window = QWidget()
+            # self.setWindowTitle(
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py:59]: Do you still need this comment: `# self.setWindowTitle(`

**Review by teweitsai — APPROVED** [review]
Looks good.

File: python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py]: Added, plus some other.

File: python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py (hunk 2 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/TopicStatusLabel.py]: Used now (was ommited before). Thanks for catching that.

**Pull request merged**